### PR TITLE
Update BROWSER_RESOLUTION_HOWTO.md

### PR DESCRIPTION
**Pull request by mvlabat**
## Content Display Protocol
- I doubt that the term "Content Display Protocol" is a good one, I don't think "displaying" has anything to do with transferring data.
- Why do we mention FTP? What use cases do we see?
- Do we want to mention technologies that we don't support at this point? This documentation can give an idea that we support Swarm, though we don't currently.

## Browser Resolution Algorithm
- We should consider reformatting this section following the practices described in [RFC 2119 - Key words for use in RFCs to Indicate Requirement Levels](https://tools.ietf.org/html/rfc2119).
- Do we assume that each distributed protocol identifies content by hash? Should we probably consider another record name, like `dweb.*.id` for instance?

### CODE REVIEW

File: BROWSER_RESOLUTION_HOWTO.md [hunk 1]
```diff
@@ -1,33 +1,35 @@
 # Browser Resolution How-to
 
-This document describe a recommended way to resolve blockchain domain within a classical HTTP Web Browser or a Dapp Browser.
-The document assumes that one having a basic understanding of how domain records are retrieved in general. See [Domain Resolution](./ARCHITECTURE.md#domain-resolution)
+This document describes a recommended way to resolve blockchain domains within a classical HTTP Web Browser or a Dapp Browser.
```
Comment (mvlabat) [inline BROWSER_RESOLUTION_HOWTO.md:3]:
```suggestion
This document describes the recommended way to resolve blockchain domains within a classical HTTP Web Browser or a Dapp Browser.
```

File: BROWSER_RESOLUTION_HOWTO.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
 ### Distributed Website Browsing
 
-1. Given a blockchain domain name configured to have a distributed network content hash (like IPFS)
-2. User enters a crypto domain within an address bar of a browser.
-3. A browser resolves a domain and gets the content hash of a domain
-4. A browser retrieves a content by the hash using a related protocol and displays the content in browser
+This scenario assumes that a blockchain domain has an dweb content identifier record configured (e.g. IPFS hash).
+
+1. A user enters the domain name into a browser address bar.
+2. A browser resolves the domain and gets the content hash of a domain.
```
Comment (DeRain) [inline BROWSER_RESOLUTION_HOWTO.md:23]: Let's rename `content hash` to `content identifier` here and below.

File: BROWSER_RESOLUTION_HOWTO.md [hunk 3]
```diff
@@ -1,33 +1,35 @@
...
-4. A browser requests and displays the content using DNS protocol and HTTP protocol.
+This scenario assumes that a blockchain domain has a DNS record configured.
+
+1. A user enters the domain name into a browser address bar.
+2. A browser resolves the domain and gets the specified DNS records.
+3. A browser requests and displays the content using DNS protocol and HTTP protocol.
 
 ### Distributed Website Browsing
 
-1. Given a blockchain domain name configured to have a distributed network content hash (like IPFS)
```
Comment (DeRain) [inline BROWSER_RESOLUTION_HOWTO.md:19]: I think `Given When Then` more structured approach to describing the scenario. It's widely used in BDD approach and can be considered as part of the Domain Specific Language.

**Comment by DeRain** [comment]
@mvlabat 

### Content Display Protocol
- I guess FTP can be the default protocol, a lot of browsers have builtin FTP client.
 - `Content Display Protocol` can be renamed to `Hypermedia protocol`.
- Agree, we should mention a protocol that is not supported right now.

### Browser Resolution Algorithm
- Great idea with RFC! I think we should consider using this in any reference description.
- I think records naming can be unique for each protocol. It's okay when different protocols have different namings and terms inside.

**Pull request merged**